doc/plugins/org.polarsys.capella.git.doc/html/GitAdapter/GitAdapterIntro.html

```diff
@@ -0,0 +1,97 @@
+<?xml version='1.0' encoding='utf-8' ?><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+		<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+		<title>GitAdapterIntro</title>
+		<link type="text/css" rel="stylesheet" href="PLUGINS_ROOT/org.polarsys.capella.doc/html/styles.css"/>
+	</head>
+	<body>
+		<h1 id="Introduction">Introduction</h1>
+		<h2 id="Introduction_to_Git">Introduction to Git</h2>
+		<p>Git is one solution to support management of versions for any kind of files.
+			<br/>
+			Git works with a central repository, and local repositories for each users.
+			<br/>
+			Basic operations when working with git are summarized below:
+		</p>
+		<h3 id="Clone_an_existing_Git_repository">Clone an existing Git repository</h3>
+		<p>
+			<img height="380" width="531" align="middle" border="0" src="images/CloneGitRepository.png"/>
+		</p>
+		<h3 id="Commit_.2F_check_out_and_push_.2F_fetch">Commit / check out and push / fetch</h3>
+		<p>
+			<img height="380" width="531" align="middle" border="0" src="images/CommitCheckOutPushFetch.png"/>
+		</p>
+		<h3 id="Working_with_several_people">Working with several people</h3>
+		<p>
+			<img height="380" width="531" align="middle" border="0" src="images/WorkingSeveralPeople.png"/>
+		</p>
+		<h2 id="Introduction_to_eGit">Introduction to eGit</h2>
+		<p>eGit is an adapter for Eclipse to support git operations directly from the Eclipse environment.&lt;/br&gt;
+			eGit provides several additional windows:</p>
+		<h3 id="Git_Repositories">Git Repositories</h3>
+		<p>To see the different repositories on which you are working and their content (branches, tags, files...)
+
+			<br/>
+
+			<img height="270" width="600" align="middle" border="0" src="images/GitRepositories.png"/>
+		</p>
+		<h3 id="Git_Staging">Git Staging</h3>
+		<p>To define which files will have to be committed and provide a commit description
```

Comment:
Some images have a .melodymodeller file in it. 
It is the old extension of the .capella file, can you update screenshot with the new extensions